.gitlab/trigger_release.yml

```diff
@@ -2,25 +2,27 @@
 # trigger_release stage
 # Contains jobs which trigger release pipelines in the datadog/agent-release-management repository.
 
+.agent_release_management_trigger:
+  stage: trigger_release
+  image: 486234852809.dkr.ecr.us-east-1.amazonaws.com/ci/datadog-agent-buildimages/deb_x64:$DATADOG_AGENT_BUILDIMAGES
```

Comment:
Any reason to use this particular image? (Nothing wrong about it, just curious)

Reply:
Not really, any image will Python 3 does the trick.